frontend/src/static/js/v_summary.js

```diff
@@ -382,7 +382,12 @@ window.vSummary = {
       });
 
       if (!this.filterGroupRepos) {
-        full[0].sort(comparator(ele => ele[this.filterSort]));
+        full[0].sort(comparator((ele) => {
+          if (typeof ele[this.filterSort] === 'string') {
+            return ele[this.filterSort].toLowerCase();
+          }
+          return ele[this.filterSort];
```

Comment:
use inline if return and do the case on presence of `toLowerCase` (always call it if possible), using `typeof` to check is generally considered to be hacky.

```
ele => {
    const field = ele[this.filterSort];
    return field.toLowerCase ? field.toLowerCase() : field;
}
```